CONTRIBUTING.md

```diff
@@ -119,10 +119,17 @@ The test script will perform following:
 
 #### Testing from the command line
 
-* npm test - runs the client tests (not including the integration tests)
-* npm run testServer - runs the server tests
-* There's nothing in place yet for running the integration tests from the command line
-
+Follow these steps to run the tests from the command line.
+* npm install
+* cd server/
+* npm install
+* cd ..
+* npm run-script compile
```

Comment:
Running these 5 commands seems common enough that I think it'd be nice to add a new "script" to package.json that runs all 5 of them - e.g. 

    npm run-script compile && npm run pretest && ...

Could call that "runClientAndServerTests" or maybe something shorter that conveys the same thing. 

I would leave the integration tests out for now since those are more fragile.